docker-compose.yml

```diff
@@ -25,6 +25,9 @@ services:
       POSTGRES_DB: iib
       POSTGRES_INITDB_ARGS: "--auth='ident' --auth='trust'"
 
+  memcached:
+    image: memcached
```

Comment:
Can you define the ports?

Reply:
Sure, sorry this was incomplete merge just to test few things up. 
I am on it, thanks.